documentation/workflows.md

```diff
@@ -0,0 +1,114 @@
+# Working on the Project
+
+Better to be explicit/specific on topics than to wait for avoidable confusion due to ambiguity.
+
+## Principles
+
+1. Changes are NEVER done directly on `main`
+    - ALWAYS branch off of the target receiving branch (i.e. `main` for the majority of work)
+    - This allows for peer review and approval
+2. All quality gates SHOULD be passed before review
+    - Linting
+    - Type-checking
+    - Isolation testing: unit, integration, functional
+    - Testing coverage thresholds
+3. All changes must be reviewed and approved by another contributor
+    - An author can not "approve" their own changes
+    - Some areas might have [code owners](../.github/CODEOWNERS.md) assigned requiring specific people to review
+
+## Development Workflows
+
+- [Starting Work](#starting-work)
+- [Pipelines](#pipelines)
+- [Changesets](#changesets)
+- [Approvals](#approvals)
+- [Local Testing](#local-testing)
+
+### Starting Work
+
+1. Pick a task from the issues/project
+    - Ask the team if anyone is working on the task
+        - Internal contributors can use private communication tools
+        - External contributors can comment in the issue directly
+    - Assign yourself to the task
+2. Create a branch off of the target receiving branch (i.e. `main`)
+    - `feat/___` - for feature work
+    - `fix/___` - for hotfix work
+    - Replace "___" with a hyphenated brief title of the task
+3. Commit changes and push the branch
+    - If changes are being made to `src/*` files or "config" for artifact output a [Changesets](#changesets) is required.
+        - `pnpm changeset`
+        - Follow the prompts to create the changeset file for your changes.
+    - Follow the conventions of [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) for the final commit of the branch.
+    - Keep the branch current; i.e. click the "Update Branch" button in the GitHub UI.
+4. Open a Pull Request to the [central repository](github.com/gohypergiant/standard-toolkit) for review
+    - Optionally add "close|closed|closes|fixed|fixes #12345" to the optional footers of a commit message; e.g. the example would close issue #12345. This is also requested as part of the PR template.
+5. Merge
+    - Authors merge PRs when all quality gates are met, reviewers have approved, and the author is ready to merge.
+    - The [Pipelines](#pipelines) will take over from here.
+    - Merging will be blocked by branch protections
```

Comment:
This line is a little confusing, perhaps remove unless you can explain what you mean here a bit more (branch protection rules are only set against `main`)